src/emulator/gdbstub/src/gdb.cpp

```diff
@@ -379,6 +488,20 @@ static std::string cmd_continue_supported(HostState &state, PacketCommand &comma
     return "vCont;c;C;s;S;t;r";
 }
 
+static std::string cmd_thread_alive(HostState &state, PacketCommand &command) {
+    const std::string content = content_string(command);
+    const int32_t thread_id = parse_hex(content.substr(1));
+
+    // Assuming a thread is removed from the map when it closes or is killed.
+    for (const auto &pair : state.kernel.threads) {
```

Comment:
Should use find(key), if this is a map

```suggestion
    if (state.kernel.threads.find(thread_id) != state.kernel.threads.end()) return "OK";
```

Reply:
Yep, sounds good.